pkg/util/util.go

```diff
@@ -43,3 +45,16 @@ func ReadMTUConfigMap(ctx context.Context, client cnoclient.Client) (int, error)
 	klog.V(2).Infof("Found mtu %d", mtu)
 	return mtu, nil
 }
+
+// IsRunningInManagedCluster returns true if the operator is running in a managed cluster.
+// It checks for the existence of the dedicated-admin namespace
+func IsRunningInManagedCluster(ctx context.Context, client cnoclient.Client) (bool, error) {
+	err := client.Default().CRClient().Get(ctx, types.NamespacedName{Name: MANAGED_CLUSTER_NAMESPACE}, &corev1.Namespace{})
+	if err != nil {
+		if errors.IsNotFound(err) {
+			return false, nil
+		}
+		return false, err
+	}
+	return true, nil
```

Comment:
we are sure we don't want to double the stored results Into the obj right? trusting this has been tested

Reply:
yep - should be fine.

Reply:
resolve.